hole/play.go

```diff
@@ -315,6 +315,12 @@ func play(
 		// is not nil. This seems to be a quirk of the Babashka interpreter
 		// that only occurs when providing code via a command line argument.
 		code += "(print)"
+	case "jelly":
+		// Prevent trivial quines. Error out and return early.
+		if hole.ID == "quine" && !strings.Contains(code, `“`) {
+			run.Stderr = "Quine in Jelly must have at least one '“' character."
```

Comment:
For starters, it must. 😝 I wouldn't know, but at least the single-byte and two-bytes nonsense I shared on Discord have been addressed. I needed three times as many bytes. 😄